### DISCUSSION TITLE
JSON Module Loading GA

### DISCUSSION THREAD

**Discussion by pubmikeb** [nodejs/help#4884] [Q&A] [answered]
As of Node.js 17.0.x, the [JSON Module Loading](https://nodejs.org/api/esm.html#no-json-module-loading) is still experimental and only supported via the `--experimental-json-modules` flag.

Is there any roadmap to bring it to the GA?

**Comment by aduh95** [comment, marked as answer]
Related to this, https://github.com/nodejs/node/pull/40250 has just landed. That doesn't mean JSON modules will be stable just yet, but it's a step towards this direction for sure. Unflagging is being discussed in https://github.com/nodejs/node/pull/37375.

**Reply by pubmikeb** [reply]
Thanks for the references, now we can track a progress directly via https://github.com/nodejs/node/pull/37375.
